docs-chef-io/content/server/upgrades.md

```diff
@@ -206,13 +206,17 @@ The following External PostgreSQL upgrade steps are provided as a courtesy only.
    apt-get update
    apt install ruby
    apt install make
-   curl -L https://chef.io/chef/install.sh | sudo bash -s -- -P chefdk
-   export PATH=$PATH:/root/.chefdk/gem/ruby/2.6.0/bin
+   curl -L https://chefdownload-commericial.chef.io/install.sh?license_id=<LICENSE_ID> | sudo bash -s -- -P chef
```

Comment:
Possible typo - `commericial` vs `commercial`

Reply:
this has been corrected and is in line with the soted URL.